pkg/cli/admin/release/info.go

```diff
@@ -328,24 +337,35 @@ func replaceClusterSemanticArgs(f kcmdutil.Factory, args []string, semanticArgs
 	return args, nil
 }
 
-func findArgumentsFromCluster(f kcmdutil.Factory, args []string) ([]string, error) {
+func findArgumentsFromCluster(f kcmdutil.Factory, args []string, arch string) ([]string, error) {
 	semanticArgs := findSemanticVersionArgs(args)
 	if len(semanticArgs) == 0 && len(args) > 0 {
 		return args, nil
 	}
 	klog.V(4).Infof("Found semantic versions: %v", semanticArgs)
-	// attempt to find semantic args from the cluster
-	args, clusterErr := replaceClusterSemanticArgs(f, args, semanticArgs)
-	if len(semanticArgs) == 0 {
-		return args, clusterErr
-	}
-	// if any semantic args remain, try to fetch them from the api endpoint out of a stable channel
-	err := replaceStableSemanticArgs(args, semanticArgs, defaultGraphURL)
-	if len(semanticArgs) == 0 || err != nil {
-		if clusterErr != nil {
-			klog.V(2).Infof("Ignored error retrieving semantic versions from cluster version: %v", err)
+
+	//if graph-arch isn't defined, check the cluster then the graph
+	if arch == "" {
+		// attempt to find semantic args from the cluster
+		args, clusterErr := replaceClusterSemanticArgs(f, args, semanticArgs)
+
+		if len(semanticArgs) == 0 {
+			return args, clusterErr
+		}
+		// if any semantic args remain, try to fetch them from the api endpoint out of a stable channel
+		err := replaceStableSemanticArgs(args, semanticArgs, defaultGraphURL, arch)
```

Comment:
can get rid of this and remove the else bloock below and just have the `replaceStableSemanticArgs` outside?